metadata.rb

```diff
@@ -6,4 +6,5 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.2.0'
 
-depends 'ark'
+depends 'ark', "= 0.9.0"
```

Comment:
Do you remember why you had to lock this to `0.9.0` ? It would probably be better to use `< 1.0.0` with a comment explaining why greater versions are problematic.